BetsApi/Controllers/BetController.cs

```diff
@@ -0,0 +1,73 @@
+﻿using AutoMapper;
+using Domain.Dto.BetQuote;
+using Domain.Dto.Bets;
+using Domain.Dto.BetRequest;
+using Microsoft.AspNetCore.Mvc;
+using Services.Interfaces;
+
+namespace BetsApi.Controllers
+{
+    [ApiController]
+    [Route("/api/[controller]")]
+    public class BetController : Controller
+    {
+        private readonly IBetsService _betService;
+        private readonly IBetQuoteService _betQuoteService;
+        private readonly IMapper _mapper;
+        private readonly IBetFacade _betFacade;
+
+        public BetController(IBetsService betService, IBetQuoteService betQuoteService, IMapper mapper, IBetFacade betFacade)
+        {
+            Console.WriteLine("Face constructorul?");
```

Comment:
remove

btw - you should use the debugger instead of logging to test stuff ...